src/cmd/src/options.rs

```diff
@@ -29,3 +32,22 @@ pub struct GlobalOptions {
     #[arg(global = true)]
     pub tokio_console_addr: Option<String>,
 }
+
+// TODO(LFC): Move logging and tracing options into global options, like the runtime options.
+/// All the options of GreptimeDB.
+#[derive(Clone, Debug, Default, Serialize, Deserialize, PartialEq)]
+#[serde(default)]
+pub struct GreptimeOptions<T> {
+    /// The runtime options.
+    pub runtime: RuntimeOptions,
```

Comment:
Why not put the runtime options to the `GlobalOptions`?

Reply:
Because `GlobalOptions` will be parsed by clap. We have to add some `#[clap]` annotations to `RuntimeOptions` fields which is not good considering it's in common-runtime crate.

Reply:
I think we could move runtime options to `StandalonOptions`/`DatanodeOptions`/`FrontendOptions`. The `GreptimeOptions` also relies on `serde(flatten)` which sometimes doesn't work well. So I prefer not introducing this struct.
https://github.com/search?q=repo%3Aserde-rs%2Fserde+flatten&type=issues

Reply:
There are already four global options repeated in `*Options`: `LoggingOptions`, `enable_telemetry`, `ExportMetricsOption` and `TracingOptions`. I ready don't like `RuntimeOptions` to be the fifth, and more to come in the future.

`serde(flatten)` is useful, and it's already used inside our codes. There are issues about it, true, and that's why I write the tests to read our example toml files into each `*Options`, to make sure it works as expected.

Putting the global options together makes our code more concise. If this PR is ok to you, I'm planning to refactor other four global options like this.

Reply:
> There are issues about it, true, and that's why I write the tests to read our example toml files into each *Options

Note that the example file contains all options but `serde(flatten)` might break `serde(default)`. I have tested some cases manually and haven't found any issues.